test/parallel/test-assert.js

```diff
@@ -280,7 +278,7 @@ testShortAssertionMessage(false, 'false');
 testShortAssertionMessage(100, '100');
 testShortAssertionMessage(NaN, 'NaN');
 testShortAssertionMessage(Infinity, 'Infinity');
-testShortAssertionMessage('', '""');
+testShortAssertionMessage('a', '"a"');
```

Comment:
Why the change here? Why not keep the original form of the test too?

Reply:
This is the formerly failing test ('' === '' is actually true and not false ;-) ).

Reply:
Huh? `'' === '""'` is false and should evaluate to false.

Reply:
Ooooh! I see now. Sorry. I misunderstood what was being tested here.